README.md

```diff
@@ -34,26 +34,29 @@ And much more...
 
 Building
 --------
-Before even building, Alure requires the OpenAL soft library (libopenal.dll, 
-libopenal.a, libopenal.dll.a) to be available and on the path of your system.  Additional 
-optional dependencies are used for playback of files, which include:
 
-* ogg: libogg.dll.a  
-* vorbis: libvorbisfile.dll.a  
-* flac: libFLAC.dll.a  
-* opusfile: libopusfile.dll.a  
-* SndFile: libsndfile.dll.a  
-* mpg123: libmpg123.dll.a  
-* PhysFS: libphysfs.dll.a  
-* dumb: libdumb.dll.a  
+#### - Dependencies -
+Before even building, Alure requires the OpenAL development files installed, for example, through 
+Creative's OpenAL SDK (available from openal.org) or from OpenAL Soft. 
 
-Some of the optional files are required for building the examples provided.  You can disable building 
-these examples if you wish by setting the cmake variable `ALURE_BUILD_EXAMPLES` to `OFF` or un-checking it in the cmake gui
+Additional 
+dependencies are required for native playback of audio files:
 
-After installation, you will want to run make install in your cmake build directory 
-in order to install OpenAL on your system
+* ogg : ogg playback
+* vorbis : ogg vorbis playback
+* flac : flac playback
+* opusfile : opus playback
+* SndFile : [see here](http://www.mega-nerd.com/libsndfile/)
+* mpg123 : mpeg audio playback
 
-Additionally you will need a C++11 compliant or above compiler to be able to build alure
+And for two examples require these dependencies to be built  
+
+* PhysFS : alure-physfs
+* dumb : alure-dumb
+
+If you fail to meet the requirements for building the examples, compilation will simply be skipped. 
```

Comment:
This goes for all dependencies, except for OpenAL itself. If any of the decoder libs aren't found, that given decoder will be skipped.